Jenkinsfile

```diff
@@ -11,35 +11,25 @@ def gitUtils = new GitUtilities()
 def utils = new Utilities()
 def bazel = new Bazel()
 
-node {
-  gitUtils.initialize()
-  bazel.setVars()
-}
-
 mainFlow(utils) {
-  if (utils.runStage('PRESUBMIT')) {
-    def success = true
-    utils.updatePullRequest('run')
-    try {
-      presubmit(gitUtils, bazel)
-    } catch (Exception e) {
-      success = false
-      throw e
-    } finally {
-      utils.updatePullRequest('verify', success)
```

Comment:
Just a general recommendation, you should put links to groovy istio-testing sources here. It'd help to demystify the Jenkins file.